src/OrleansTestingHost/TestingSiloHost.cs

```diff
@@ -661,13 +677,29 @@ public static SiloHandle StartOrleansSilo(TestingSiloHost host, Silo.SiloType ty
 
             // Load initial config settings, then apply some overrides below.
             ClusterConfiguration config = new ClusterConfiguration();
-            if (options.SiloConfigFile == null)
+            try
             {
-                config.StandardLoad();
+                if (options.SiloConfigFile == null)
+                {
+                    config.StandardLoad();
```

Comment:
Ok, I'll do it in a few days, since I'd like to see how it breaks.
I do agree that what's currently there is not very good. But since it's going away and the only reason we are keeping it is to not break current users, I would not break them further unnecessarily. If they are already using the config file, then silently reverting them to the Localhost programmatic config will be completely unexpected.

I'll see what I can do about it next week. If you do it before then, let me know so I rebase my other PR that it's currently built after these commits.